stages/org.osbuild.oscap.autotailor

```diff
@@ -49,6 +52,20 @@ SCHEMA = """
         "type": "array",
         "items": { "type": "string" },
         "description": "The rules to deselect from the base OpenSCAP profile"
+      },
+      "overrides": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "required": ["var", "value"],
+          "properties": {
+            "var": { "type": "string" },
+            "value": {
+              "type": [ "string", "integer" ]
```

Comment:
Can those values only integers or are there overrides that also take e.g. string values?

Reply:
It can be both, I realised all the tests are integers. But it can be strings too. 

It got me thinking, so I'm just verifying if we can get away with just using string representations of the ints

Reply:
Yeah we can just use strings. I'll update the PR.

Reply:
Thanks, I think it's fine as is I was reading this a bit too shallow, let's merge it and get you unblocked.

Reply:
Fwiw, I added a tiny unit test around this in https://github.com/osbuild/osbuild/compare/main...mvo5:autotailor-overrides-test?expand=1 that I will open once this is merged (we want new stages to have some lightweight tests around them now but the policy is quite new so we are happy to write them as needed for now).

Reply:
Awesome, thanks for the tests :)

Funnily enough, it seems that a string is fine in most cases... but I found a weird bug where SSH would break if you used a `string` instead of an `int` for the `sshd_idle_timeout_value`.

Reply:
Thanks! I wonder how this happens, do you have the details of the inputs you passed and how it broke? Just for my own curiosity :)

Reply:
I took a break from it because it was driving me a little mad. I'll take a look at it again this morning

Reply:
It's really, really weird. The inputs are the only thing that is different.

The output of the command is identical, the output of the tailoring file is identical. But I can SSH into one and not the other.